### ISSUE TITLE
tojpg helper chokes on 'JPEG' extension

### ISSUE THREAD

**Issue by eaton** [jarednova/timber#722]
This bit me recently — it doesn't recognize that 'JPEG' files are the same as 'JPG', so it falls through to the default case and assumes the file is a PNG. Errors ensue, and it results in empty white image files.

**Comment by eaton** [comment]
FYI, the following kludgey change on line 64 of `/lib/image/timber-image-operation-tojpg.php` got things working for me locally:

```
		if ( $ext == 'png' ) {
		    return imagecreatefrompng($filename);
		}
		if ( ($ext == 'jpeg') || ($ext == 'jpg') ) {
		    return imagecreatefromjpeg($filename);
		}
```

**Comment by jarednova** [comment]
I've seen kludgier things in my day. Could you submit a PR?

**Comment by eaton** [comment]
Will do, thanks!

**Comment by jarednova** [comment]
Closed by #733
